docs/installation/prepare/rpi-setup.md

```diff
@@ -184,10 +184,6 @@ For these changes to take effect, you need to restart your Raspberry Pi if it ha
 sudo reboot
 ```
 
-## Next Steps
```

Comment:
do you think it might make more sense to remove the second one? i.e. have troubleshooting come after the next steps?

Reply:
ok done! I also changed the wifi capitalization across doc to be WiFi